src/components/DebuggerV2/handlers.tsx

```diff
@@ -156,5 +156,24 @@ export function getHandlers(platformSDK: LedgerLiveApi): MethodHandler[] {
         },
       ],
     },
+    {
+      id: "signMessage",
+      name: "Sign a message",
+      description:
+        "Display a Native UI and have the user signing a message using his Hardware Wallet",
+      handler: platformSDK.signMessage.bind(platformSDK),
+      inputs: [
+        {
+          name: "account",
+          component: AccountSelector,
+        },
+        {
+          name: "message",
+          component: JSONTextArea,
+          modifier: (value: any) => value,
```

Comment:
The `JSONTextArea` component could be turned in a basic `TestArea` component and the `mode` (like `json`) could be passed as a prop

Reply:
Indeed